src/_posts/2018-01-09-beam-a-look-back.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: post
+title:  "Apache Beam: A Look Back at 2017"
+date:   2018-01-09 00:00:01 -0800
+excerpt_separator: <!--more-->
+categories: blog
+authors:
+  - ianand
+  - jbonofre
+---
+
+On January 10, 2017, Apache Beam got [promoted]({{ site.baseurl }}/blog/2017/01/10/beam-graduates.html)
+as a Top-Level Apache Software Foundation project. It was an important milestone
+that validated the value of the project, legitimacy of its community, and
+heralded its growing adoption. In the past year, Apache Beam has been on a
+phenomenal growth trajectory, with significant growth in its community and
+feature set. Let us walk you through some of the notable achievements.
+
+<!--more-->
+
+## Use cases
+
+First, lets take a glimpse at how Beam was used in 2017. Apache Beam being a
+unified framework for batch and stream processing, enables a very wide spectrum
+of diverse use cases. Here are some use cases that exemplify the versatility of
+Beam.
+
+![Grid computer and numerical simulations, batch ETL and ML pre-processing,
+  streaming ETL, IoT and real-time anomaly detection, real-time mobile game
+  session personalization](
+  {{ "/images/beam_versatility_use_cases.svg" | prepend: site.baseurl }})
```

Comment:
I only just now notice that there's a blog directory inside images, I guess the image should go somewhere in there.